tools/functional-tester/etcd-tester/tester.go

```diff
@@ -176,6 +176,10 @@ func (tt *tester) checkConsistency() (failed bool, err error) {
 	if !ok || err != nil {
 		plog.Printf("%s checking current revisions failed [revisions: %v]", tt.logPrefix(), revs)
 		failed = true
+		cerr := tt.cleanup()
```

Comment:
Do you mean 

```go

func (tt *tester) checkConsistency() (failed bool, err error) {
	tt.cancelStressers()
	defer func() {
		if err == nil {
			err = tt.startStressers()
		}
	}()
```

?

We still need to run stressers even with the errors for next rounds?